GUI/MainInstall.cs

```diff
@@ -434,6 +434,7 @@ private void UpdateRecommendedDialog(Dictionary<string, List<string>> mods, bool
                     "The following modules have been recommended by one or more of the chosen modules:";
                 RecommendedModsListView.Columns[1].Text = "Recommended by:";
                 RecommendedModsToggleCheckbox.Text = "(De-)select all recommended mods.";
+                RecommendedModsToggleCheckbox.Checked=true;
```

Comment:
Problem in this line: The recommended and suggested page are the same. They are just renamed.
Therefore the Checkbox is also on in the suggested page, although it should be off.